src/views/auth/Login.vue

```diff
@@ -0,0 +1,90 @@
+<script setup lang="ts">
+import { loginSession } from '@/apis/sessions'
+import { OpenMessage } from '@/utils/openComponents'
+import { onMounted, reactive, ref } from 'vue'
+import { useRouter } from 'vue-router'
+import OButton from './components/Button.vue'
+import oCard from './components/Card.vue'
```

Comment:
Why are there both uppercase O and lowercase o?

Reply:
fixed!